### DISCUSSION TITLE
ComicShannsMono: Almost impossible to tell the difference between lowercase `y` and uppercase `Y`

### DISCUSSION THREAD

**Discussion by antoniosarosi** [ryanoasis/nerd-fonts#1338] [General]
I started using ComicShannsMono Nerd Font, which I downloaded from the [downloads page](https://www.nerdfonts.com/font-downloads). Recently I noticed that it's very hard to distinguish lowercase `y` from uppercase `Y`, as the only thing that changes is the position of the letter. Example in VSCode:

![2023-08-15-203601_246x187_scrot](https://github.com/ryanoasis/nerd-fonts/assets/54243067/fdf07ba4-bc6d-42f7-9634-1b8916d44d9b)

This is the content of the screenshot:

```
y = "y"
Y = "Y"
y = "Y"
Y = "y"
```

Is this problem related to [https://github.com/jesusmgg/comic-shanns-mono](https://github.com/jesusmgg/comic-shanns-mono)? Should I open an issue in that repo instead?

**Comment by Finii** [comment]
> Is this problem related to https://github.com/jesusmgg/comic-shanns-mono? Should I open an issue in that repo instead?

Well, Nerd Fonts just inserts the icons into existing fonts. So the Latin letters are just how we receive them.
So yes, that would need to be reported over there.

But then, all the Comic fonts are interrelated, do the all suffer this? I believe the `yY` is a characteristic of that fonts and so _it's a feature not a bug_ :grimacing: 

https://github.com/shannpersand/comic-shanns
https://github.com/dtinth/comic-mono-font
https://en.wikipedia.org/wiki/Comic_Sans

Your example looks really bad. But that improves if there are letters beside the `Y` and in normal code I have no problems to distinguish the two.
